File: test_archives/cpp_functions/batch3/function1.cpp
#include <string> 
#include <vector> 
#include <numeric>

using namespace std; 

float function1(string string1, int int1, float float1, vector<float> list1) { 
    string my_string = string1; 
    for (int i = 0; i < int1 - 1; i++) { 
        my_string += string1; 
    } 
    float my_float = float1 * list1.size(); 
    vector<float> my_list; 
    for (float x : list1) { 
        my_list.push_back(x * float1); 
    } 
    float my_total = my_string.length() + my_float; 
    return my_total; 
}

File: test_archives/py_functions/batch3/function1.py
def function1(string1, int1, float1, list1): 
    my_string = string1 * int1  # my_string is a string
    my_float = float1 * len(list1)  # my_float is a float
    my_list = [x*float1 for x in list1]  # my_list is a list of floats
    my_total = my_string + my_float  # my_total is a float
    return my_total  # returns one singular final value